java/code/src/com/redhat/rhn/domain/server/test/ServerTest.java

```diff
@@ -248,8 +248,8 @@ public void testOsDoesNotSupportsOSImageBuilding() throws Exception {
                 ServerConstants.getServerGroupTypeSaltEntitled(),
                 ServerFactoryTest.TYPE_SERVER_MINION);
         s.setOs("SLES");
-        s.setRelease("11.4");
-        assertFalse(s.doesOsSupportsContainerization());
+        s.setRelease("10.4");
```

Comment:
I don't understand, on your issue you said:

> Hint: SLES11 does supports OSImageBuilding

But here you are changing to SLE10.

Can you explain?